src/widgets/current-refinements/current-refinements.tsx

```diff
@@ -4,86 +4,74 @@ import { h, render } from 'preact';
 import cx from 'classnames';
 import CurrentRefinements from '../../components/CurrentRefinements/CurrentRefinements';
 import connectCurrentRefinements, {
-  Item,
   CurrentRefinementsConnectorParams,
-  CurrentRefinementsRendererOptions,
 } from '../../connectors/current-refinements/connectCurrentRefinements';
 import {
   getContainerNode,
   createDocumentationMessageGenerator,
 } from '../../lib/utils';
 import { component } from '../../lib/suit';
-import { WidgetFactory, Renderer } from '../../types';
-
-type CurrentRefinementsCSSClasses = {
-  root: string | string[];
-  list: string | string[];
-  item: string | string[];
-  label: string | string[];
-  category: string | string[];
-  categoryLabel: string | string[];
-  delete: string | string[];
-};
+import { WidgetFactory } from '../../types';
 
-export type CurrentRefinementsComponentCSSClasses = {
-  [TClassName in keyof CurrentRefinementsCSSClasses]: string;
-};
+export type CurrentRefinementsCSSClasses = {
+  /**
+   * CSS class to add to the root element.
+   */
+  root?: string | string[];
 
-export type CurrentRefinementsWidgetParams = {
   /**
-   * The CSS Selector or `HTMLElement` to insert the widget into.
+   * CSS class to add to the list element.
    */
-  container: string | HTMLElement;
+  list?: string | string[];
+
   /**
-   * The CSS classes to override.
+   * CSS class to add to the each item element.
    */
-  cssClasses?: Partial<CurrentRefinementsCSSClasses>;
+  item?: string | string[];
+
+  /**
+   * CSS class to add to the label element.
+   */
+  label?: string | string[];
+
   /**
-   * The attributes to include in the widget (all by default).
-   * Cannot be used with `excludedAttributes`.
-   *
-   * @default `[]`
+   * CSS class to add to the category element.
    */
-  includedAttributes?: string[];
+  category?: string | string[];
+
   /**
-   * The attributes to exclude from the widget.
-   * Cannot be used with `includedAttributes`.
-   *
-   * @default `['query']`
+   * CSS class to add to the categoryLabel element.
    */
-  excludedAttributes?: string[];
+  categoryLabel?: string | string[];
+
   /**
-   * Receives the items, and is called before displaying them.
-   * Should return a new array with the same shape as the original array.
-   * Useful for mapping over the items to transform, and remove or reorder them.
+   * CSS class to add to the delete element.
    */
-  transformItems?: (items: Item[]) => any;
+  delete?: string | string[];
 };
 
-type CurrentRefinementsRendererWidgetParams = {
-  container: HTMLElement;
-  cssClasses: CurrentRefinementsComponentCSSClasses;
+export type CurrentRefinementsComponentCSSClasses = {
```

Comment:
It'd make more sense if this lived in the component file instead (`CurrentRefinements.tsx`).

Reply:
done.